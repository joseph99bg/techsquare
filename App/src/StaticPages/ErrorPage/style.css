.error-page h1 {
    text-align: center;
    font-size: 10vw;
    line-height: 10vw;
    margin: 5vh 0 1vh 0;
    font-family: 'Orbitron';
    color: #707ed6;
}
.error-page h3 {
    text-align: center;
    font-size: 50px;
    font-family: 'Montserrat';
    color: #707ed6;
}
.error-page .btn-holder {
    text-align: center;
}
.error-page button {
    font-size: 30px;
    color: #FFF;
    font-family: 'Montserrat';
    background: #707ed6;
    padding: 20px 40px;
    border: 3px solid #707ed6;
    cursor: pointer;
}
.error-page button:hover {
    background: transparent;
    color: #707ed6;
}