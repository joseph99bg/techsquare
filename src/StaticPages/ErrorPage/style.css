.error-page h1 {
    text-align: center;
    font-size: 10vw;
    line-height: 10vw;
    margin: 5vh 0 1vh 0;
    font-family: 'Orbitron';
    color: #707ed6;
}
.error-page h3 {
    text-align: center;
    font-size: 50px;
    font-family: 'Montserrat';
    color: #707ed6;
}