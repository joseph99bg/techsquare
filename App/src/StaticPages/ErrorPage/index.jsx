import React from 'react'
import { Link } from 'react-router-dom'
import './style.css'

function ErrorPage() {
    return (
        <div className="error-page">
            <h1>404</h1>
            <h3>Page Not Found</h3>
            <div className="btn-holder">
                <Link to="/">
                    <button><i className="fas fa-home"></i> Return to Home</button>
                </Link>
            </div>
        </div>
    )
}

export default ErrorPage