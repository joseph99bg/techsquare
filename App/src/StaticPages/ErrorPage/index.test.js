import React from 'react'
import ErrorPage from './index'
import renderer from 'react-test-renderer'

jest.mock("react-router-dom", () => ({
    Link: "Link"
}));

describe('ErrorPage component', () => {
    test('should render correctly', () => {
        const component = renderer.create(
            <ErrorPage />
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})