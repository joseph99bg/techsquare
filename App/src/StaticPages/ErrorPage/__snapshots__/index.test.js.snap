// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage component should render correctly 1`] = `
<div
  className="error-page"
>
  <h1>
    404
  </h1>
  <h3>
    Page Not Found
  </h3>
  <div
    className="btn-holder"
  >
    <Link
      to="/"
    >
      <button>
        <i
          className="fas fa-home"
        />
         Return to Home
      </button>
    </Link>
  </div>
</div>
`;
